lib/org/openpsa/invoices/invoice/pdf.php

```diff
@@ -63,9 +67,14 @@ public function get_button_options() : array
         ];
     }
 
-    public function render_and_attach() : midcom_db_attachment
+    public function render_and_attach(string $kind = null) : midcom_db_attachment
     {
-        $client_class = midcom_baseclasses_components_configuration::get('org.openpsa.invoices', 'config')->get('invoice_pdfbuilder_class');
+        if ($kind == null) {
+            $client_class = midcom_baseclasses_components_configuration::get('org.openpsa.invoices', 'config')->get('invoice_pdfbuilder_class');
+        } else {
+            $client_class = midcom_baseclasses_components_configuration::get('org.openpsa.invoices', 'config')->get('invoice_pdfbuilder_class_' . $kind);
```

Comment:
invoice_pdfbuilder_class_reminder needs to be added to config.inc, otherwise there's be a warning in the log file.

Also invoice_pdfbuilder_reminder_class would be a more logical name

Reply:
is the current version okay? i'm not sure if the changes at the config.inc are correct.

Reply:
config has `invoice_pdfbuilder_reminder_class`. Here you build the string `invoice_pdfbuilder_class_reminder`. So, no

Reply:
looks like this is outdated. i actually changed it to invoice_pdfbuilder_reminder_class